### DISCUSSION TITLE
UCAN Users List

### DISCUSSION THREAD

**Discussion by expede** [ucan-wg/spec#98] [Ideas]
I've had a few folks ask for a list of projects that use UCAN today (e.g. as a way of helping gain adoption in their org). Alongside a cookbook, this feels like it could be useful and relatively low effort. You do see this kind of thing for programming languages and frameworks. Does anyone have thoughts or concerns?

**Comment by bmann** [comment]
We should file issues in the UCAN community website for this stuff. Cookbook should live there too. 

And: have a kick off of those who want to help work on this.
